src/validate-reducer/index.js

```diff
@@ -0,0 +1,23 @@
+export {validateReducer}
+
+const EXIT_EARLY = 'EXIT_EARLY'
+const CONTINUE = 'CONTINUE'
+
+validateReducer.EXIT_EARLY = EXIT_EARLY
+validateReducer.CONTINUE = CONTINUE
```

Comment:
>> See the tests :-)

Spot on.

*I wish, someday GitHub would allow user to reply instead of another comment.*